src/librustdoc/markdown.rs

```diff
@@ -143,10 +143,9 @@ pub fn test(mut options: Options, diag: &errors::Handler) -> i32 {
                                        options.linker, options.edition, options.persist_doctests);
     collector.set_position(DUMMY_SP);
     let codes = ErrorCodes::from(UnstableFeatures::from_environment().is_nightly_build());
-    let res = find_testable_code(&input_str, &mut collector, codes);
-    if let Err(err) = res {
-        diag.span_warn(DUMMY_SP, &err.to_string());
-    }
+
+    find_testable_code(&input_str, &mut collector, codes);
```

Comment:
Why those changes?

Reply:
While I was making the changes to `find_testable_code`, I noticed that the error case isn't actually possible. The code attempts to handle a missing codeblock end event but I don't think `pulldown-cmark` can actually emit that sequence of events.  Furthermore, there isn't a test case in the compiler that emits the error message.

So, I removed the error case, changed the signature from `Result` to `()`, and updated all the call-sites to remove the error emitting code.

Reply:
I see, good catch then!

Reply:
https://github.com/rust-lang/rust/issues/60319